wagn/features/attach.feature

```diff
@@ -41,4 +41,19 @@ Feature: Conflict
     And I press "Submit"
     Then I should see an image of size "large" and type "jpg"
 
+  Scenario: uploading a file as a field in a form
+    When I go to  "/new Cardtype"
+    And I fill in "card_name" with "complicated_card"
+    And I press "Submit"
+    And I create HTML card "complicated card+*type+*structure" with content "{{+image}}{{+description}}"
+    And I go to "/new complicated_card"
+    And I fill in "card_name" with "Vignesh has a complicated relationship"
+    And I wait until ajax response done
```

Comment:
That's better than just waiting a fixed time. I suggest you replace the "wait a sec" statements in my scenarios above with that.